server/src/main/java/org/opensearch/extensions/ExtensionsOrchestrator.java

```diff
@@ -86,18 +85,18 @@ public static enum RequestType {
     };
 
     private final Path extensionsPath;
-    final Set<DiscoveryExtension> extensionsSet;
-    Set<DiscoveryExtension> extensionsInitializedSet;
+    final List<DiscoveryExtension> extensionsList;
+    List<DiscoveryExtension> extensionsInitializedList;
     TransportService transportService;
     ClusterService clusterService;
 
     public ExtensionsOrchestrator(Settings settings, Path extensionsPath) throws IOException {
         logger.info("ExtensionsOrchestrator initialized");
         this.extensionsPath = extensionsPath;
         this.transportService = null;
-        this.extensionsSet = new HashSet<DiscoveryExtension>();
-        this.extensionsInitializedSet = new HashSet<DiscoveryExtension>();
-        this.clusterService = null;
```

Comment:
We need `clusterService` just like `transportService ` above

Reply:
Looks like I accidentally deleted an extra line